Index.md

```diff
@@ -1,63 +1,57 @@
-| [Text-Analyzer](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Text%20Analyzer)                       | The Text Analyzer analyze the text entered by the user. It will give information such as word count, character count, number of sentences, number of spaces, number of punctuators used in the text. |
-| [Tic-Tac-Toe Game](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Tic-Tac-Toe%20Game)                       | Simple Tic-Tac-Toe Game written in HTML, CSS, and JavaScript  |
-| [TO-DO List](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/To-Do%20List)                                   | Simple TO-DO List written in HTML, CSS, and JavaScript   |
-| [Weather-App](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Weather-App)                                   | Simple Weather app written in HTML, CSS, and JavaScript using the APIs for fetching weather and geolocation information                                                                                                              |
-| [News Webapp](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/flask_news_project)                            | A web app for searching news made using flask                                                                                                                                                                                        |
-| [Score Keeper](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Score-Keeper)                                  |A web Page where user can keep a count of their scores and will tell which player won.|
-| [Image Filter App](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Image-Filter-App)                                                                                                            |An Awesome Image Filter App written in Html,Css,Javascript and CamanJS.|
-| [Form Validation](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/form-validation)                         |A basic sign up page with all the validations in javascript.   |     
-| [Age Calculator](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/age-calculator)                           | A website which can be used to calculate age of a person in days,months and years . |
-| [BMI Calculator](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/bmi-calculator)                           | A basic website which can be used to calculate body mass index |
-| [Glassmorphism Animation Form](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Glassmorphism-Form)      |A basic login page with Glassmorphism Animations |       

```

Comment:
Some of the projects have been removed. Please put them back. Also, don't try to copy paste the entire thing. Try updating your fork first before adding your project details and then make the PR. This will avoid conflicts.